xml/System.IO.Compression/ZipFileExtensions.xml

```diff
@@ -199,7 +200,7 @@
         <exception cref="T:System.IO.DirectoryNotFoundException">
           <paramref name="sourceFileName" /> is invalid (for example, it is on an unmapped drive).</exception>
         <exception cref="T:System.IO.PathTooLongException">In <paramref name="sourceFileName" />, the specified path, file name, or both exceed the system-defined maximum length. For example, on Windows-based platforms, paths must not exceed 248 characters, and file names must not exceed 260 characters.</exception>
-        <exception cref="T:System.IO.IOException">The file specified by <paramref name="sourceFileName" /> cannot be opened.</exception>
+        <exception cref="T:System.IO.IOException">The file specified by <paramref name="sourceFileName" /> cannot be opened, or is too large to be updated (current limit is [`Int32.MaxValue`](https://docs.microsoft.com/dotnet/api/system.int32.maxvalue)).</exception>
```

Comment:
you'll probably need the see href tag here and not a markdown link (I assume)

Reply:
That syntax worked.  I just checked on the review site.